Rtp/RtpClient.Methods.cs

```diff
@@ -358,7 +358,7 @@ public System.Collections.Generic.IEnumerable<Rtcp.RtcpReport> PrepareReports(Tr
         internal protected /*virtual*/ int SendReceiversReport(TransportContext context, bool force = false)
         {
             //Determine if the ReceiversReport can be sent.
-            if (Common.IDisposedExtensions.IsNullOrDisposed(this) | Common.IDisposedExtensions.IsNullOrDisposed(context)  //If the context is disposed
+            if (Common.IDisposedExtensions.IsNullOrDisposed(this) || Common.IDisposedExtensions.IsNullOrDisposed(context)  //If the context is disposed
```

Comment:
I suppose this is a good fix, the other way was intentionally checking in parallel. but this is good I suppose.